packages/api-aco/src/record/record.gql.ts

```diff
@@ -83,6 +85,15 @@ export const searchRecordSchema = new GraphQLSchemaPlugin<AcoContext>({
         }
     `,
     resolvers: {
+        SearchRecord: {
+            id: async parent => {
+                const { id } = parseIdentifier(parent.id);
+                return removeAcoRecordPrefix(id);
+            },
+            entryId: async parent => {
```

Comment:
I would remove it, since it has the same value of `id`. 
If we decide to keep it, we should add it to `SearchRecord` type